<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Artis-MBP.fios-router.home" ignored="0" name="DataDrivenFramework" tests="1" failures="1" timestamp="2021-04-01T18:21:47 EDT" time="5.438" errors="0">
  <testcase name="loginAsManager" time="5.412" classname="org.example.testcases.BankManagerLoginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.example.testcases.BankManagerLoginTest.loginAsManager(BankManagerLoginTest.java:22)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- loginAsManager -->
</testsuite> <!-- DataDrivenFramework -->
